data/campaigns/Eastern_Invasion/scenarios/14_The_Drowned_Plains.cfg

```diff
@@ -0,0 +1,2037 @@
+    #         y={FOGY} # (I couldn't get this working with radius and
+    #  [not])
+    #     [/lift_fog]
+#enddef
+        {SCROLL_TO 68 42}
+        {UNSHROUD_UNFOG 68 42  68,67,67,68,69  41,42,43,43,43}
+        [delay]
+            time=700
+        [/delay]
+
+        {SCROLL_TO 17 5}
+        {UNSHROUD_UNFOG 17 5  17,18,18,17,16,16  4,4,5,6,5,4}
+        [delay]
+            time=700
+        [/delay]
+
+        {SCROLL_TO 7 50}
+        {UNSHROUD_UNFOG 7 50  7,8,8,7,6,6  49,49,50,51,50,49}
+        [delay]
+            time=700
+        [/delay]
+
+        [message]
+            speaker=Dacyn
+            message= _ "Owaec, do not be so hasty. Much of the horde that conquered this place now marches on Weldyn, but even what little lingers is surely more than a match for our weary band."
+        [/message]
+        [message]
+            speaker=Gweddry
+            message= _ "I agree, yet all is strangely quiet. Perhaps the remaining undead lie dormant? If we take time to capture villages and rebuild our forces, we may yet be able to reclaim this place."
+        [/message]
+
+        [objectives]
+            side=1
+            [objective]
+                description= _ "Defeat all enemy leaders"
+                condition=win
+            [/objective]
+            [objective]
+                {OPTIONAL_OBJECTIVE_CAPTION}
+                description= _ "Capture villages and build up an army before attacking."
+                condition=win
+            [/objective]
+            [objective]
+                {OPTIONAL_OBJECTIVE_CAPTION}
+                description= _ "Loot Addogin’s smugglers’ caches."
+                condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL caches_active equals yes}
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Death of Gweddry, Dacyn, or Owaec"
+                condition=lose
+            [/objective]
+            {TURNS_RUN_OUT}
+            [gold_carryover]
+                bonus=yes
+                carryover_percentage=40
+            [/gold_carryover]
+            [note]
+                description= _ "Enemy leaders are idle until first sighted, but will then abandon their keep and attack you."
+            [/note]
+            #
+            #              [note]
+            #                 description= _ "Ghosts will not attack you."
+            #             [/note]
+        [/objectives]
+
+        [event]
+            name=side turn end,moveto,attack,recruit,recall,select # as soon as you do anything at all
+            [message]
+                speaker=Hahid al-Ali
+                message= _ "On a quest of searing vengeance? Owaec my friend, what you need is my elixir of ulfserker fury! One taste of this bitter red ambrosia and you’ll fight until you drop, with <b><i>unlimited melee attacks</i></b> and the <b><i>berserk</i></b> melee special!"
```

Comment:
Comma after 'Owaec'.

Reply:
This kind of construct is definitely valid in some situations (like if "my friend" were replaced by "dear", though obviously that doesn't fit in this context), but I'm not sure if it's valid in this situation… certainly safe to add a comma though.